app/view_objects/apply_invalid_data_view.rb

```diff
@@ -83,4 +84,13 @@ def populate_degree_fields(degree_fields)
       field_and_values.keys
     end
   end
+
+  def get_field_name(field)
+    case field
+    when "uk_degree" || "non_uk_degree"
+      "degree_type"
```

Comment:
```suggestion
I18n.t("views.apply_invalid_data_view.unrecognised_field_text", invalid_field: field.to_s.humanize.upcase_first)
```

You can just reuse this i18n call (maybe put into a method) and it'll return `Degree type` which will get parametirized properly without the gsubbing.

Reply:
Done 👍